app/src/main/java/com/mobileproto/hireddit/hireddit/MainActivity.java

```diff
@@ -62,5 +55,33 @@ public boolean onOptionsItemSelected(MenuItem item)
         return super.onOptionsItemSelected(item);
     }
 
+    private void switchFragment(Fragment f)
+    {
+        FragmentTransaction transaction = manager.beginTransaction();
+        transaction.replace(R.id.container, f).addToBackStack(null);
+        transaction.commit();
+    }
+
+    private void switchFragment(Fragment f, int customAnimationIn, int customAnimationOut)
```

Comment:
try not to name both switchfragment (maybe switchFragmentAnimation or something)